packages/admin/admin/src/common/toolbar/ToolbarBreadcrumbs.tsx

```diff
@@ -0,0 +1,295 @@
+// TODO: Add theming and support for `sx` and `slotProps` when #1809 is merged: https://github.com/vivid-planet/comet/pull/1809
+import { ChevronDown, ChevronRight, ChevronUp } from "@comet/admin-icons";
+import {} from "@emotion/react";
+import { ButtonBase, css, ListItemText, Menu, MenuItem, styled, Theme, Typography } from "@mui/material";
+import React from "react";
+import { Link as RouterLink, LinkProps as RouterLinkProps } from "react-router-dom";
+
+import { useStackApi } from "../../stack/Api";
+import { useObservedWidth } from "../../stack/breadcrumbs/utils"; // TODO: Update import when merged: https://github.com/vivid-planet/comet/pull/1755
```

Comment:
Merged today, please update

Reply:
I'd prefer to do this in a separate PR, since this is planned for another sprint.